build/arm/linker/linker_freertos_task_symbols.ld

```diff
@@ -0,0 +1,4 @@
+__rtos_capture_start = .;
```

Comment:
How about `.data` objects as well?

Reply:
There is nothing non-zero-initialized in there:
```
 .bss.xTimerTaskHandle
                0x0000000020006ac0        0x4 ../../../build/target/freertos/platform-26-m//libfreertos.a(timers.o)
                0x0000000020006ac4                __rtos_capture_data = .
 *tasks.o(.data .data*)
 *timers*.o(.data .data*)
                0x0000000020006ac4                __rtos_capture_end = .
```

We can't really coalesce things into two different locations with different initialization schemes.

This is also not going to be changed in FreeRTOS as there is a `PRIVILEGED_DATA` functionality for MPUs that relies on the fact that everything is pretty much in bss as well.